client/src/routes/settings/+page.svelte

```diff
@@ -157,6 +221,108 @@
     </Actions>
   </Dialog>
 
+  <!--Delete Account Button and Prompt-->
+  <Dialog
+    bind:open={showDeletionPrompt}
+    aria-labelledby="confirm-account-deletion-title"
+    aria-describedby="deletion-confirmation-content"
+  >
+    <Title id="confirm-account-deletion-title">Delete Account</Title>
+    <Content id="deletion-confirmation-content">Are you sure?</Content>
+    <Actions>
+      <Button>
+        <Label>Cancel</Label>
+      </Button>
+      <Button on:click={verifyDelete}>
+        <Label>Delete Account</Label>
+      </Button>
+    </Actions>
+  </Dialog>
+  <!--If user signed in too long ago, redirect them to sign in-->
+  {#if deleteErr !== ""}
```

Comment:
what does this if statment do?

Reply:
it opens a dialog that tells the user that they need to sign in again and they click ok and it redirects them back to the sign in  page and logs them out